### PR TITLE
DX-2889 Add Media API Integration Tests

### PR DESCRIPTION
**Pull request by ajrice6713**

### CODE REVIEW

**Review by brianluisgomez — DISMISSED** [review]

File: src/test/java/org/openapitools/client/api/MediaApiTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    ApiClient defaultClient = Configuration.getDefaultApiClient();
+    HttpBasicAuth Basic = (HttpBasicAuth) defaultClient.getAuthentication("Basic");
+    private final MediaApi api = new MediaApi(defaultClient);
+
+    private String mediaPath = "src/test/java/org/openapitools/client/fixtures/";
+    private String mediaFile = "java_cat.jpeg";
+    private static UUID uuid;
+    private String mediaId = JAVA_VERSION + "_" + RUNNER_OS + "_" + uuid + "_" + mediaFile;
+    private File media = new File(mediaPath + mediaFile);
+    private String content_type = "image/jpeg";
```
Comment (matthewkmartin) [inline src/test/java/org/openapitools/client/api/MediaApiTest.java:37]:
```suggestion
    private String contentType = "image/jpeg";
```

File: src/test/java/org/openapitools/client/api/MediaApiTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    private String cache_control = "no-cache";
```
Comment (matthewkmartin) [inline src/test/java/org/openapitools/client/api/MediaApiTest.java:38]:
```suggestion
    private String cacheControl = "no-cache";
```

File: src/test/java/org/openapitools/client/api/MediaApiTest.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    @Test
+    @Order(1)
+    public void uploadMedia() throws ApiException {
+        Basic.setUsername(BW_USERNAME);
+        Basic.setPassword(BW_PASSWORD);
+
+        ApiResponse<Void> response = api.uploadMediaWithHttpInfo(BW_ACCOUNT_ID, this.mediaId, this.media,
+                this.content_type,
```
Comment (matthewkmartin) [inline src/test/java/org/openapitools/client/api/MediaApiTest.java:52]:
```suggestion
                this.contentType,
```

File: src/test/java/org/openapitools/client/api/MediaApiTest.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                this.cache_control);
```
Comment (matthewkmartin) [inline src/test/java/org/openapitools/client/api/MediaApiTest.java:53]:
```suggestion
                this.cacheControl);
```

**Review by matthewkmartin — CHANGES_REQUESTED** [review]

**Review by matthewkmartin — APPROVED** [review]

File: src/test/java/org/openapitools/client/api/MediaApiTest.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    public void uploadMedia() throws ApiException {
+        Basic.setUsername(BW_USERNAME);
+        Basic.setPassword(BW_PASSWORD);
+
+        ApiResponse<Void> response = api.uploadMediaWithHttpInfo(BW_ACCOUNT_ID, this.mediaId, this.media,
+                this.contentType,
+                this.cacheControl);
+
+        assertThat(response.getStatusCode(), is(204));
+        System.out.println(this.mediaId);
```
Comment (ckoegel) [inline src/test/java/org/openapitools/client/api/MediaApiTest.java:56]: is there a reason for all of these printlns?

Comment (ajrice6713) [reply src/test/java/org/openapitools/client/api/MediaApiTest.java]: No - removed

File: pom.xml [hunk 6]
```diff
@@ -394,4 +410,4 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
-</project>
+</project>
```
Comment (brianluisgomez) [inline pom.xml:413]:
```suggestion
</project>

```

**Review by ckoegel — DISMISSED** [review]

**Review by brianluisgomez — APPROVED** [review]

**Review by brianluisgomez — APPROVED** [review]

**Review by ckoegel — APPROVED** [review]

**Pull request merged**